version.txt

```diff
@@ -1 +1 @@
-0.5.1
```

Comment:
I think this is the only place still using this file: [Makefile#L88](https://github.com/fmatoss/wysihtml5/blob/a63822eec3a1f0ab26231c641d60ba6f8ec44a41/Makefile#L88)

We could remove by getting the current version from `package.json` with `grep`